hack/vendor.sh

```diff
@@ -1,28 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Looks like you removed a bit too much here; the `hack/vendor.sh` script itself should probably still be there (or Jenkins to be updated);


```
[2023-10-23T17:06:17.709Z] + docker run --rm -t --privileged -v /home/ubuntu/workspace/moby_PR-46695/.git:/go/src/github.com/docker/docker/.git --name docker-pr2 -e DOCKER_EXPERIMENTAL -e DOCKER_GITCOMMIT=d6cbc6b37d407d5bc8573bcfbbcf803b8fcb91e4 -e DOCKER_GRAPHDRIVER -e TEST_FORCE_VALIDATE -e VALIDATE_REPO=https://github.com/moby/moby.git -e VALIDATE_BRANCH=20.10 docker:d6cbc6b37d407d5bc8573bcfbbcf803b8fcb91e4 hack/validate/vendor
[2023-10-23T17:06:17.969Z] hack/validate/vendor: line 14: ./hack/vendor.sh: No such file or directory
script returned exit code 127
```

Reply:
Whoops, missed that one. Those pesky horizontal scroll bars...